lib/cisco_node_utils/cmd_ref/portchannel_global.yaml

```diff
@@ -75,5 +74,5 @@ rotate:
   default_value: 0
 
 symmetry:
-  _exclude: [N7k, N3k, N5k, N6k]
+  _exclude: [N7k, N5k, N6k]
```

Comment:
* From now on when we have a platform support change we need to also update the 'caveats' table in the Puppet README.md file: https://github.com/cisco/cisco-network-puppet-module/blob/develop/README.md#type-cisco_portchannel_global